tests/testDegenerateTemplate.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+#
+# Copyright 2008-2016  AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import print_function
+import unittest
+import numpy as np
+
+import lsst.utils.tests
+import lsst.afw.detection as afwDet
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.pex.logging as pexLogging
+from lsst.meas.deblender.baseline import deblend
+import lsst.meas.algorithms as measAlg
+
+root = pexLogging.Log.getDefaultLog()
+root.setThreshold(pexLogging.Log.DEBUG)
+
+# Quiet some of the more chatty loggers
+pexLogging.Log(root, 'lsst.meas.deblender.symmetrizeFootprint',
+               pexLogging.Log.INFO)
+pexLogging.Log(root, 'lsst.meas.deblender.symmetricFootprint',
+               pexLogging.Log.INFO)
+pexLogging.Log(root, 'lsst.meas.deblender.getSignificantEdgePixels',
+               pexLogging.Log.INFO)
+pexLogging.Log(root, 'afw.Mask',
+               pexLogging.Log.INFO)
+
```

Comment:
Should be using `lsst.log` now.  See, e.g. https://github.com/lsst/meas_deblender/commit/3cf000259809b44fc2ecb636a8eb16ac76931169 for how to convert.